### DISCUSSION TITLE
Scrollable Frame

### DISCUSSION THREAD

**Discussion by Abdulsamipy** [TomSchimansky/CustomTkinter#259] [General]
Hello am trying to create a frame where users can generate multiple entries with a button, the issue is when multiple entries are created the window is unable to scroll down, is it possible if I can make the Comment Frame scrollable only? 

Image 1 :Before creating multiple entries  , Image 2 : after creating multiple entries

![Screenshot 2022-07-12 205032](https://user-images.githubusercontent.com/58009877/178534584-5cef6146-0255-434b-88b3-436b215bd937.png)
![Screenshot 2022-07-12 204828](https://user-images.githubusercontent.com/58009877/178534636-d158fa79-7f25-4df1-8d98-5e6a7f3eb1d6.png)

**Comment by TomSchimansky** [comment]
Look here: https://blog.teclado.com/tkinter-scrollable-frames/
But maybe I will add a scrollable frame in the future...

**Comment by mq1** [comment]
https://blog.teclado.com/tkinter-scrollable-frames/ doesn't seem to integrate well with customtkinter, a CTkScrollableFrame class would be appreciated 🚀

**Comment by mq1** [comment]
@TomSchimansky If it can be useful, i made https://github.com/mq1/ice-launcher/blob/main/ice_launcher/components/scrollable_frame.py
